src/compiler/program.ts

```diff
@@ -888,13 +888,13 @@ namespace ts {
 
         function computeCommonSourceDirectory(sourceFiles: SourceFile[]): string {
             let commonPathComponents: string[];
-            forEach(files, sourceFile => {

```

Comment:
Interesting...I wonder why this was `forEach` before. It seems like a labelled `for` would have been more obvious.

Regardless, it's convenient because it makes your change smaller.